libcontainer/configs/cgroup_unix.go

```diff
@@ -12,6 +12,8 @@ const (
 
 type Cgroup struct {
 	Name string `json:"name"`
+	// indicates that this is an externally passed, fully created cgroup
+	ExternalCgroup bool `json:"external_cgroup"`
```

Comment:
We shouldn't have a bool for this. If there is a path passed, then we use it. Resource may or may not be empty.